### PR TITLE
add python3 mummi, maestrowf

### PR DESCRIPTION
**Pull request by jmoon1506**
added all changes for python3

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-maestrowf/package.py [hunk 1]
```diff
@@ -11,17 +11,16 @@ class PyMaestrowf(PythonPackage):
...
-    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.2.tar.gz"
+    git      = "https://github.com/LLNL/maestrowf.git"
 
-    version('1.1.4dev1.1', git='https://github.com/LLNL/maestrowf.git', tag='1.1.4dev1.1')
+    version('1.1.4dev1.1', tag='1.1.4dev1.1')
+    version('flux-dev', branch='bugfix/flux_broker')
+    version('develop', branch='develop')
+    version('1.1.4', tag='v1.1.4') 
+    version('1.1.2', tag='v1.1.2')
+    version('flux-sched', branch='bugfix/flux_broker')
```
Comment (FrankD412) [inline var/spack/repos/builtin/packages/py-maestrowf/package.py]: This one points to the same branch as `flux-dev`, so it's redundant.

**Review by FrankD412 — COMMENTED** [review]
Overall, this looks fine to me. I just left a comment about Maestro's package.

File: var/spack/repos/builtin/packages/py-maestrowf/package.py [hunk 2]
```diff
@@ -11,17 +11,16 @@ class PyMaestrowf(PythonPackage):
        simulation studies."""
 
     homepage = "https://github.com/LLNL/maestrowf/"
-    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.2.tar.gz"
+    git      = "https://github.com/LLNL/maestrowf.git"
```
Comment (bhatiaharsh) [inline var/spack/repos/builtin/packages/py-maestrowf/package.py]: Ok, so after having thought about it, i think i am sure this is not a good idea. by changing the default download method to git, we're forcing all users to set up proper ssh keys. whereas, by leaving it as path to tar gets around the problem, and makes the tool completely public!

here and below, we should have sha hash of all the downloadable releases. use git only for the branches and bug-fixes, like @FrankD412  had before.

**Comment by jmoon1506** [comment]
Okay ready for merge

**Pull request merged**